tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java

```diff
@@ -253,4 +256,38 @@ public void testFallbackMethodWithArgsSuccess() {
         Assert.assertEquals(fallbackClient.getCounterForInvokingServiceE(), 2, "The execution count should be 2 (1 retry + 1)");
         
     }
+
+    /**
+     * Test that a Fallback service can work alone, without other annotations.<br>
+     * <p>
+     * ServiceA fallback is driven by a class implementing FallbackHandler. ServiceA is only called once.
+     */
+    @Test
+    public void testStandaloneClassLevelFallback() {
```

Comment:
The annotation `Fallback` is method level annotation only as it is less useful to be on class level.

Reply:
ah. You meant `Fallback` handler. please change the method name to be testStandaloneHandlerFallback